src/CTA.Rules.Actions/ActionHelpers/ServerConfigMigrate.cs

```diff
@@ -137,7 +137,8 @@ private void HandleRequestFiltering(XElement requestFilteringElement)
         private void HandleHttpRedirect(XElement httpRedirectElement)
         {
             StringBuilder sb = new StringBuilder("new RewriteOptions()");
-            bool isRedirectionEnabled = bool.Parse(GetAttributeValue(httpRedirectElement, Constants.Enabled));
```

Comment:
Do we know if `bool.Parse()` throws if the argument is null?

Reply:
yes, bool.Parse() has a check in it that throws an exception if the argument is null. I am assuming that I do not need to check if the `XElement` input is null, because that pattern (not checking if the input arg is null) is repeated across all of the `Handle` functions.

Reply:
Unless it is possible for `httpRedirectElement` to not be null, but for `GetAttributeValue` to still return null...

Reply:
Wait but I'm already handling it for that case...

Reply:
Correct. We ensure the XElement object is not null before calling any of these `Handle*` methods. 

For future reference, it is generally easier to test/maintain code if parameter validation is handled by the receiving method rather than by the caller. But there is no hard and fast rule saying one way or the other needs to be done.

Reply:
> Wait but I'm already handling it for that case... 
Very possible, my mistake.